src/System.IO.FileSystem/src/System/IO/Enumeration/EnumerationOptions.cs

```diff
@@ -0,0 +1,78 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.IO.Enumeration
+{
+    public class EnumerationOptions
+    {
+        /// <summary>
+        /// For internal use. These are the options we want to use if calling the existing Directory/File APIs where you don't
+        /// explicitly specify EnumerationOptions.
+        /// </summary>
+        internal static EnumerationOptions Compatible => new EnumerationOptions { MatchType = MatchType.Dos };
```

Comment:
Why do we create a new one each time rather than using a singleton? i.e. why is this:
```C#
=>
```
rather than
```C#
{ get; } = 
```
?

Is the options instance exposed in some way through the public API such that someone could change that singleton?  Or do we change it internally?

Reply:
Mistake on my part. Fixing.